lib/jekyll-last-modified-at/determinator.rb

```diff
@@ -3,49 +3,47 @@
 module Jekyll
   module LastModifiedAt
     class Determinator
-      attr_reader :site_source, :page_path
+      @repo_cache = {}
+      @path_cache = {}
```

Comment:
This is actually a last modified time cache keyed on paths.  `@last_mod_cache` perhaps?